dags/glam_glean_imports.py

```diff
@@ -2,13 +2,15 @@
 Desktop ETL for importing glean data into GLAM app
 """
 
-from datetime import datetime, timedelta
+import datetime
```

Comment:
Is there a specific reason why you decided to change the import? I saw you use `datetime.date`, could also just add `date` to the existing import?